lighthouse-core/report/html/templates.html

```diff
@@ -149,108 +149,49 @@
   </div>
 </template>
 
-
-<!-- Lighthouse header -->
-<template id="tmpl-lh-heading">
+<!-- Lighthouse topbar -->
+<template id="tmpl-lh-topbar">
   <style>
-    :root {
-      --report-header-overlap-top: 30px;
-    }
-    .lh-header-bg {
-      background-color: var(--header-bg-color);
-      height: var(--report-header-height);
-      left: 0;
-      position: absolute;
+    .lh-topbar {
+      position: fixed;
       top: 0;
-      width: 100%;
-      will-change: transform;
-      contain: strict;
-    }
-    .lh-lighthouse {
-      position: absolute;
-      bottom: -4px;
+      left: 0;
       right: 0;
-      opacity: 1;
-      transform-origin: bottom right;
-      will-change: transform, opacity;
-    }
-    .lh-header {
-      max-width: var(--report-width);
-      margin: 0 auto;
-      height: var(--report-header-height);
-      position: relative;
-    }
-    .lh-metadata {
-      flex: 1 1 0;
-      padding: calc(var(--section-padding) / 2);
-      padding-left: var(--section-indent);
-      line-height: 20px;
-      color: var(--report-header-color);
-      z-index: 1;
-      position: relative;
-      max-width: 60%;
-    }
-    .lh-metadata__results {
-      text-overflow: ellipsis;
-      white-space: nowrap;
-      overflow: hidden;
-    }
-    .lh-metadata__url {
-      color: currentColor;
-    }
-
-    .lh-product-info, .lh-toolbar__metadata {
-      align-items: center;
-      white-space: nowrap;
-      color: #5F6369;
-      display: flex;
-      font-size: calc(var(--body-font-size) * 0.9);
-      margin-left: var(--section-indent);
-      opacity: 0;
-      transform: translateY(-50%);
-      will-change: opacity;
-    }
-    .lh-product-info__icon {
-      height: 20px;
-      margin-right: var(--default-padding);
-    }
-    .lh-toolbar {
-      height: 50px;
-      position: absolute;
-      top: 25px;
-      will-change: transform;
+      z-index: 1000;
       display: flex;
-      width: calc(100% - 70px);  /* give room for export */
+      align-items: center;
+      height: var(--topbar-height);
+      background-color: var(--topbar-bg);
+      padding: var(--topbar-padding);
     }
-    .lh-toolbar__metadata {
-      overflow: hidden;
-      text-overflow: ellipsis;
-      width: 100%;
+
+    .lh-topbar__logo {
+      width: var(--icon-size);
+      height: var(--icon-size);
+      background-image: var(--logo-url);
+      background-size: cover;
     }
-    .lh-toolbar__url {
-      color: currentColor;
-      display: block;
-      white-space: nowrap;
-      margin-right: 2px;
+
+    .lh-topbar__url {
+      margin-left: 8px;
```

Comment:
seems like you could reuse --topbar-padding here.

Reply:
done